guide/index.md

```diff
@@ -4,17 +4,11 @@ title: Getting Started | Guide
 
 # Getting Started
 
-::: tip [Learn PWA](https://web.dev/learn/pwa/)
-Progressive Web Apps (PWAs) are web application built and enhanced with modern APIs to deliver enhanced capabilities, reliability, and installability while reaching anyone, anywhere, on any device, all with a single codebase.
+Progressive Web Apps (PWAs) are web applications built and enhanced with modern APIs to deliver enhanced capabilities, reliability, and installability while reaching anyone, anywhere, on any device&mdash;all with a single codebase.
```

Comment:
what's `&mdash;`  here?

Reply:
Oh, it's just "&mdash;", the [em dash](https://www.thepunctuationguide.com/em-dash.html). I generally use the HTML entity `&mdash;` because it's easier to manage, especially in monospaced fonts.